co2calculator/calculate.py

```diff
@@ -204,6 +211,9 @@ def calc_co2_bus(
     :return: Total emissions of trip in co2 equivalents, distance of the trip
     :rtype: tuple[float, float]
     """
+    # NOTE: fuel_type is only 'diesel' - that makes the argument quite superfluous
+    # NOTE: vehicle_rage 'local' fails with IndexError
```

Comment:
good hint, we should probably add 'local' to all 'Linienbus' in the emission factor table https://github.com/pledge4future/co2calculator/blob/dev/data/emission_factors.csv